sdk/appconfiguration/app-configuration/tsconfig.json

```diff
@@ -13,7 +13,8 @@
     "forceConsistentCasingInFileNames": true,
     "outDir": "./dist-esm",
     "importHelpers": true,
-    "newLine": "LF"
+    "newLine": "LF",
+    "lib": ["dom", "es5", "es6", "es7", "esnext"]
```

Comment:
Per our [guidelines](https://azure.github.io/azure-sdk/typescript_design.html#ts-config-lib) we don't want to set the `lib` field in tsconfig.json. Instead you can use the triple-slash reference directives in the files that need them.

Reply:
Changed over to a reference directive instead.